framework-cloud/framework-git/src/main/java/org/kie/cloud/git/GitProviderService.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.cloud.git;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.kie.cloud.git.constants.GitConstants;
+import org.kie.cloud.git.github.GitHubGitProviderFactory;
+import org.kie.cloud.git.gitlab.GitLabGitProviderFactory;
+import org.kie.cloud.git.gogs.GogsGitProviderFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GitProviderService {
+    private final Map<String, GitProviderFactory> providerFactories;
+
+    private static final Logger logger = LoggerFactory.getLogger(GitProviderService.class);
```

Comment:
I would probably suggest LOGGER, since it is a private static final field. But I am not aware of code conventions in this particular repository.

Reply:
We use logger with small letters everywhere in this test suite. I will keep it this way.

Reply:
Sure, np.